src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -121,7 +121,8 @@ public class RedisHubLifetimeManager<THub> : HubLifetimeManager<THub>, IDisposab
 
                 if (groupMessage.Action == GroupAction.Remove)
                 {
-                    if (!await RemoveGroupAsyncCore(groupMessage.ConnectionId, groupMessage.Group))
+                    var connection = _connections[groupMessage.ConnectionId];
+                    if (connection == null || !(await RemoveGroupAsyncCore(connection, groupMessage.Group)))
                     {
                         // user not on this server
```

Comment:
update the comment? `RemoveGroupAsyncCore` returns false if the group does not exist.